docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -128,7 +128,25 @@ specific environment, you may need to choose different CIDRs.
 So, once you have decided which type of network to create, following the
 appropriate instructions for one of *a)*, *b)*, *c)* or *d)*.
 
-#### a. Networking using Calico IPAM in a non-cloud environment
+>#### NOTE: Docker Networks now Allow Specifying IP on Container
```

Comment:
I don't think this is important for the mainline flow of this document and makes this section, which is already long, even longer.

I'd move this to the bottom (maybe above IPv6) and add it as it's own section on adding statically assigned IPs.